scripts/upload-docs-build-gh.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# exit with nonzero exit code if anything fails
+set -e
+
+# Get the current branch name
+CURRENT_BRANCH=`git symbolic-ref -q --short HEAD`
+
+# Only run this script if we're on `master`
```

Comment:
Oh can we not do branch builds like on docspot with GH?

Reply:
Nope, there's only "one" github pages deployment